package.json

```diff
@@ -45,7 +45,7 @@
 		"dom-react": "2.2.1",
 		"dom-scroll-into-view": "1.2.1",
 		"element-closest": "2.0.2",
-		"equivalent-key-map": "0.2.0",
+		"equivalent-key-map": "0.2.1",
```

Comment:
We can remove it from here, it is used only in `data` package.

Reply:
For some reason, I didn't understand, removing it break the unit tests (but only on travis)

Reply:
Notice I removed it and than added it back in the commits